projects/1.16/assets/glassential/glassential/lang/zh_cn.json

```
@@ -1 +1,19 @@
-{}
\ No newline at end of file
+{
+    "block.glassential.glass_dark": "暗色玻璃",
+    "block.glassential.glass_dark_ethereal": "暗色空灵玻璃",
+    "block.glassential.glass_dark_ethereal_reverse": "反转暗色空灵玻璃",
+    "block.glassential.glass_ethereal": "空灵玻璃",
+    "block.glassential.glass_ethereal_reverse": "反转空灵玻璃",
+    "block.glassential.glass_ghostly": "幽灵玻璃",
+    "block.glassential.glass_light": "发光玻璃",
+    "block.glassential.glass_redstone": "红石玻璃",
+	
+    "tooltip.glassential.dark": "阻光",
+    "tooltip.glassential.ethereal": "对玩家而言不是固体",
+    "tooltip.glassential.ethereal_reverse": "仅对玩家而言是固体",
+    "tooltip.glassential.ghostly": "对实体而言不是固体",

```

```suggestion
    "tooltip.glassential.ethereal": "玩家不能穿过此玻璃（固体方块）",
    "tooltip.glassential.ethereal_reverse": "只有玩家可以穿过此玻璃（非固体方块）",
    "tooltip.glassential.ghostly": "所有实体都无法穿过此玻璃（固体方块）",
```
https://minecraft.fandom.com/wiki/Solid_block

---

```suggestion
    "tooltip.glassential.ethereal": "玩家可穿行",
    "tooltip.glassential.ethereal_reverse": "仅玩家无法穿行",
    "tooltip.glassential.ghostly": "所有实体皆可穿行",
```

---

@n0099 你怎么看这个译法

---

good